CMakeLists-implied-options.txt

```diff
@@ -345,6 +347,10 @@ if (LWS_WITH_LIBEVENT)
 	set(LWS_WITH_LIBEVENT 1)
 endif()
 
+if (LWS_WITH_SDEVENT)
+	set(LWS_WITH_SDEVENT 1)
+endif()
```

Comment:
The conditional and the set variable are both the exact same. Is this check even useful?

Reply:
I copied this, not sure whether its a bug (could be), or it is there to ensure that anything "defined" is an integer 1 while anything undefined remains undefined (similar to what https://stackoverflow.com/questions/14751973/what-is-in-c does)

Need to check...

Reply:
I forwarded the question to https://github.com/warmcat/libwebsockets/issues/2130, waiting for an answer.

Reply:
Removed. (The maintainers found this pointless as well.)